aggregate-limitations.rst

```diff
@@ -22,7 +22,7 @@ Aggregate Limitations
    If you have many media attachments or more than 10MB of media attachments, :doc:`Form Uploader <form-uploader>` provides a quick and easy means to upload forms into ODK Aggregate.
 
 
-- ODK Aggregate 1.4.7 and earlier use a deprecated `backends` technology. Google may terminate support for that at any time with little warning. You should consider upgrading to 1.4.8.  
+- ODK Aggregate 1.4.7 and earlier use a deprecated *backends* technology. Google may terminate support for that at any time with little warning. You should consider upgrading to 1.4.8.  
```

Comment:
I think that backend is a simple adjective here, and should have:

 - no 's'
 - no markup

> ...use a deprecated backend technology.